redirects.csv

```diff
@@ -16,26 +16,18 @@ https://forms.crossroads.net/*,https://${env:CRDS_APP_DOMAIN}/forms/:splat,302!
 https://embedint.crossroads.net/*,https://int.crossroads.net/giving/,302!
 https://embeddemo.crossroads.net/*,https://demo.crossroads.net/giving/,302!
 https://embed.crossroads.net/*,https://www.crossroads.net/giving/,302!
-/,${env:CRDS_UNIFIED_DOMAIN}h,200! Role=user
 /,${env:CRDS_UNIFIED_DOMAIN},200!
+/home,${env:CRDS_UNIFIED_DOMAIN}home,200!
```

Comment:
Changing this to `/home` may not be necessary now that you're updates in crdschurch/crds-unified#327 are coming more into focus. 

If you do update this path, you'll need to update the post registration/login redirect in crdschurch/crds-components to reflect this new path _and_ make sure that there is a clear means by which the user can manually navigate back to the logged in homepage. 

Due to these considerations, I'd consider dropping this change and leaving the logged in homepage where it is.